megamek/src/megamek/common/templates/InfantryTROView.java

```diff
@@ -185,7 +185,7 @@ private void addWeaponNotes(List<String> notes) {
             notes.add(String.format(Messages.getString("TROView.InfantryNote.SingleFieldGun"),
                     fieldGuns.get(0).getName(), shots, (int) fieldGuns.get(0).getTonnage(inf)));
         }
-        if (inf.getSecondaryN() > 1) {
+        if (inf.getSecondaryWeapon() != null) {
```

Comment:
checks "secondaryN" but then looks at getSecondaryWeapon(), which results in an NPE when secondaryN > 1 but there's no secondary weapon.

Reply:
Do you know what unit was causing that? This fix is fine, but it suggests another problem somewhere else.

Reply:
No idea. I was trying to customize an infantry unit with the goal of installing field guns, so (roughly):
- pull up 'customize in mek lab'
- actually click on the 'mek lab' because there's a bug with the command center where it goes to the infirmary tab instead
- swap motive type to tracked
- attempt to use the field gun drop down, fail

Reply:
Oh yeah, it was a customized ER support laser unit with some kind of sealed environment suit.